src/mainmenu.cpp

```diff
@@ -11,7 +11,6 @@ MainMenu::MainMenu(QWidget *parent) :
 {
     auto app = KiwixApp::instance();
     setStyle(new MenuProxyStyle());
-
```

Comment:
I like this extra line to logically separate blocks of code. As we don't modify any code in this file, let's just exclude it from this PR.